pandas/tests/series/test_reductions.py

```diff
@@ -114,3 +116,61 @@ def test_validate_stat_keepdims():
     )
     with pytest.raises(ValueError, match=msg):
         np.sum(ser, keepdims=True)
+
+
+@pytest.mark.parametrize(
```

Comment:
It seems that most other idxmin/max and argmin/max tests are in pandas/tests/reductions/test_reductions.py. Could you maybe move them there?

Reply:
Moved tests to appropriate locations in /tests/reductions/test_reductions.py. Thanks.